### DISCUSSION TITLE
Optional tool for adding a color background to elements

### DISCUSSION THREAD

**Discussion by mystifiedmunchkin** [awcodes/filament-tiptap-editor#319] [Ideas]
Hi again, I know most web designers don't like the users messing around with adding too many styles to their site, totally understandable, but seeing the color picker is there for text, just wondering if it might be in any future release to let people chose a background color for stuff?
That said, this plugin rocks 🙂 thanks for all the hard work, making and maintaining it 😎

**Comment by awcodes** [comment]
Background colors for things like what?

**Comment by mystifiedmunchkin** [comment]
Table cells or grid divs would be cool, but otherwise just the text? The point being, if the users change a text color, a site's dark mode will override a pale background and make the colored text less legible...

**Reply by awcodes** [reply]
Wouldn't that be more about implementation of output and less about the plugin /admin side of editing content.

**Reply by awcodes** [reply]
Not saying you're wrong, just not totally understanding the use case.
